lib/view/info_view.dart

```diff
@@ -18,6 +19,13 @@ class InfoView extends StatelessWidget {
   }
 
   final topCards = <Widget>[
+    ReusableCard(
+      title: 'WH Novel Coronavirus information',
+      color: _cardColor,
+      height: _cardHeight,
+      elevation: _cardElevation,
+      fallback: () => BaseUrlLauncher.launchWithBrowser(whCoronavirusInfoURL),
```

Comment:
I think you might have misunderstood, and have made this unnecessarily complicated. All your code does, is call `if (await canLaunch(url)) await launch(url)`, which already existed in `contact_launcher.dart`. No need to turn one class into three, just use existing `contact_launcher.dart`.

Reply:
It is bit confusing to use `ContactLauncher` to launch coronavirus information. And `contact_launcher.dart` has functions like `makeCall` and `sendEmail` which are not related to launching coronavirus information. That's why I created a `BaseUrlLauncher` which has the base functions to launch a URL.